dmriprep/run.py

```diff
@@ -519,7 +519,7 @@ def drop_outliers_fn(in_file, in_bval, in_bvec, drop_scans):
         from nipype.utils.filemanip import fname_presuffix
 
         img = nib.load(op.abspath(in_file))
-        img_data = img.get_fdata()
```

Comment:
@richford and specific reason for `get_fdata`? it kills my python when I run it.

Reply:
Just following along with [nibabel's getting started page](http://nipy.org/nibabel/gettingstarted.html). But I now see (from [here](http://nipy.org/nibabel/images_and_memory.html)) that we could also use:
```
img_data = np.asarray(img.dataobj)
```
It looks like [`get_data` is deprecated](http://nipy.org/nibabel/reference/nibabel.dataobj_images.html?highlight=get_data#nibabel.dataobj_images.DataobjImage.get_data) and will be removed around April 2020.

Reply:
hm, weird. When you ran the function, did you run it from your python env on your mac, or through the docker image? I wonder why it's not working for me. I'm going to merge this for now, but will make an issue about resolving this at a later time